src/main/features/response/form/models/timelineBreakdown.ts

```diff
@@ -0,0 +1,66 @@
+import { ValidateNested } from 'class-validator'
+
+import { Serializable } from 'models/serializable'
+import { TimelineRow } from 'response/form/models/timelineRow'
+
+export const INIT_ROW_COUNT: number = 4
+
+export class TimelineBreakdown implements Serializable<TimelineBreakdown> {
+  readonly type: string = 'breakdown'
+
+  @ValidateNested({ each: true })
+  rows: TimelineRow[]
+
+  constructor (rows: TimelineRow[] = TimelineBreakdown.initialRows()) {
+    this.rows = rows
+  }
+
+  static initialRows (): TimelineRow[] {
+    let rows: TimelineRow[] = []
```

Comment:
this could be nicer with a reduce I think.

Reply:
rather than having a mutable collection here

Reply:
What would tell about:

```
static initialRows (): TimelineRow[] {
    return Array(INIT_ROW_COUNT).fill(TimelineRow.empty())
}
```

?